Support/GRDB.h

```diff
@@ -7,4 +7,9 @@ FOUNDATION_EXPORT double GRDB_VersionNumber;
 FOUNDATION_EXPORT const unsigned char GRDB_VersionString[];
 
 #import <GRDB/GRDB-Bridging.h>
-#import <GRDB/sqlite3.h>
+
+#if SQLITE_HAS_CODEC
+    #import <SQLCipher/sqlite3.h>
```

Comment:
I'd like to see a `#ifdef COCOAPODS` guard the inclusion of `<SQLCipher/sqlite3.h>`.